src/store/dictionary.ts

```diff
@@ -63,8 +63,8 @@ export const dictionaryStore = createPartialStore<DictionaryStoreTypes>({
       { state, dispatch },
       { surface, pronunciation, accentType, priority }
     ) {
-      // 同期処理により、一つのエンジンだけに登録しても、他のエンジンにも反映される。
-      // むしろ全てのエンジンに登録処理をするとUUIDが合わない。
+      // 同期処理（SYNC_ALL_USER_DICT）により、一つのエンジンだけに登録しても、他のエンジンにも反映される。
+      // むしろ全てのエンジンに登録処理をするとUUIDが合わず、追加した単語が同期処理で再登録される。
```

Comment:
なるほどです、同じ単語IDで登録したいから同期処理をするんですね！
であれば、AのためにBするというコメントを書くと更にわかりやすいかなと思いました！
（別に変えなくても大丈夫だと思います）

```suggestion
      // 同じ単語IDで登録するために、１つのエンジンで登録したあと全エンジンに同期する。
```